pallets/liquidation-pools/src/lib.rs

```diff
@@ -638,7 +638,7 @@ impl<T: Config> Pallet<T> {
 		let oracle_price = T::PriceSource::get_underlying_price(pool_id).ok_or(Error::<T>::InvalidFeedPrice)?;
 		let balance_ratio = Self::liquidation_pool_data_storage(pool_id).balance_ratio;
 		// Liquidation pool ideal balance in USD: liquidity_pool_balance * balance_ratio * oracle_price
```

Comment:
```suggestion
		// Liquidation pool ideal balance in USD: liquidity_pool_borrow_usd * balance_ratio * oracle_price
```
Also fix comments for this function in line 635: `liquidity_pool_borrow_usd`

Reply:
**In** tests for liquidation-pools, you need to correct comments:
```
/*
			Liquidity Pools **borrows** (in assets): [500_000, 1_000_000, 1_500_000, 2_000_000]
			Liquidity Pools balances (in USD): [500_000, 2_000_000, 7_500_000, 20_000_000]
			Ideal balances 0.2 * liquidity_pool_balance (in USD): [100_000, 400_000, 1_500_000, 4_000_000]

			Liquidation Pools balances (in assets): [400_000, 300_000, 200_000, 100_000]
			Liquidation Pools balances (in USD): [400_000, 600_000, 1_000_000, 1_000_000]

			Sales list (in assets): [(DOT, BTC, 300_000$), (KSM, BTC, 200_000$)]

			*/
```